### PR TITLE
Fix/refactoring flobject.py

### PR DESCRIPTION
**Pull request by hpohekar**
Removed code duplications in command, query and get_cls.

### CODE REVIEW

**Review by mkundu1 — APPROVED** [review]

**Review by seanpearsonuk — APPROVED** [review]
Nice work, @hpohekar. Thanks.

**Comment by seanpearsonuk** [comment]
@hpohekar Please can you create a Fluent PR with the same changes since flobject.py also exists in Fluent for the PyConsole feature.

**Comment by hpohekar** [comment]
Hello @seanpearsonuk I have created a Fluent PR.

File: src/ansys/fluent/core/solver/flobject.py [hunk 1]
```diff
@@ -802,60 +801,55 @@ def get_cls(name, info, parent=None):
...
+                    doc += f"    {ccls.__name__} : {th}\n"
+                    doc += f"        {ccls.__doc__}\n"
+
                 while ccls_name in taboo:
                     if i > 0:
                         ccls_name = ccls_name[: ccls_name.rfind("_")]
                     i += 1
                     ccls_name += f"_{str(i)}"
                 ccls.__name__ = ccls_name
                 # pylint: disable=no-member
```
Comment (dnwillia-work) [inline src/ansys/fluent/core/solver/flobject.py:833]: Is this line still needed?  You could check it with pylint...

File: src/ansys/fluent/core/solver/flobject.py [hunk 2]
```diff
@@ -802,60 +801,55 @@ def get_cls(name, info, parent=None):
...
+
+        children = info.get("children")
+        if children:
+            taboo.add("child_names")
+            cls.child_names = []
+            _process_cls_names(children, cls.child_names)
+
         commands = info.get("commands")
         if commands:
             cls.command_names = []
```
Comment (dnwillia-work) [inline src/ansys/fluent/core/solver/flobject.py:846]: I think it's a bit more Pythonic these days to write `cls.command_names = list()`

File: src/ansys/fluent/core/solver/flobject.py [hunk 3]
```diff
@@ -641,18 +641,23 @@ class Map(SettingsBase[DictStateType]):
     """A Map object represents key-value settings."""
 
 
+def _get_new_keywords(obj, kwds):
+    newkwds = {}
```
Comment (dnwillia-work) [inline src/ansys/fluent/core/solver/flobject.py:645]: Personally I like doing it this way:  `newkwds = dict()`

File: src/ansys/fluent/core/solver/flobject.py (hunk 2 above)
Comment (hpohekar) [reply src/ansys/fluent/core/solver/flobject.py:846]: Hi Dan, 
I will follow these instructions now onwards.
Thank you.

File: src/ansys/fluent/core/solver/flobject.py (hunk 3 above)
Comment (hpohekar) [reply src/ansys/fluent/core/solver/flobject.py:645]: Hi Dan, 
I will follow these instructions now onwards.
Thank you.

File: src/ansys/fluent/core/solver/flobject.py (hunk 1 above)
Comment (hpohekar) [reply src/ansys/fluent/core/solver/flobject.py:833]: Hello @dnwillia-work,

Should I create new pyfluent and Fluent PR to implement following changes?  

1. removal of " # pylint: disable=no-member "
2. cls.command_names = list()
3. newkwds = dict()

Comment (dnwillia-work) [reply src/ansys/fluent/core/solver/flobject.py:833]: Yeah sure, when you have time, I did not realize you had already merged it.

**Pull request merged**